src/commands/lambda/uninstrument.ts

```diff
@@ -18,6 +18,11 @@ import {
 import {getUninstrumentedFunctionConfigs, getUninstrumentedFunctionConfigsFromRegEx} from './functions/uninstrument'
 import {FunctionConfiguration} from './interfaces'
 import {requestAWSCredentials, requestChangesConfirmation, requestFunctionSelection} from './prompt'
+import {
+  renderError,
+  renderNoAWSCredentialsFound,
+  renderNoDefaultRegionSpecifiedError,
+} from './renderers/common-renderer'
 import * as renderer from './renderers/instrument-uninstrument-renderer'
```

Comment:
Is there a way to have these imports as one?
```suggestion
import * as commonRenderer from './renderers/common-renderer'
import * as renderer from './renderers/instrument-uninstrument-renderer' // maybe change `renderer` to something else?
```

This is so we don't have tons of imports from renderers. 
We might need to change the name of whole imports to be descriptive.